timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -27,100 +25,96 @@ def test_analyzer(self):
                 MockDataStore)
     def test_same_session(self):
         """Test multiple events in the same session are allocated correctly."""
-        with mock.patch.object(
-                self.analyzer_class,
-                'event_stream',
-                return_value=_create_mock_event(
-                    0,
-                    2,
-                    time_diffs=[self.analyzer_class.max_time_diff_micros / 2])):
-            index = 'test_index'
-            sketch_id = 1
-            analyzer = self.analyzer_class(index, sketch_id)
-            message = analyzer.run()
-            self.assertEqual(
-                message,
-                'Sessionizing completed, number of session created: 1')
-
-            ds = MockDataStore('test', 0)
-            event1 = (ds.get_event('test_index', '0', stored_events=True))
-            self.assertEqual(event1['_source']['session_id'],
-                             {analyzer.session_type: 1})
-            # checking event with id '101' as 100 events have been inserted
-            # as 'padding' (see _create_mock_event())
-            event2 = (ds.get_event('test_index', '101', stored_events=True))
-            self.assertEqual(event2['_source']['session_id'],
-                             {analyzer.session_type: 1})
+        index = 'test_index'
+        sketch_id = 1
+        analyzer = self.analyzer_class(index, sketch_id)
+        analyzer.datastore.client = mock.Mock()
```

Comment:
The mocking of the datastore.client is needed because in the event_stream method, which is called in the sessionizer.run() method, are updated the client's indices. MockDataClient cannot be used, because it doesn't suports the needed funtionality. I didn't implement it because I had to mock IndicesClient class function, located in the virtual enviorment, and I didn't know how. My solution for the problem was to use Mock class, but I am open to another ideas.